contracts/FIFOMarket.sol

```diff
@@ -118,22 +118,16 @@ contract FIFOMarket is
       recipient != address(0),
       "FIFOMarket: Cannot mint to the 0 address"
     );
-    // todo verify this can only be invoked by the nori contract
+    // TODO: verify this can only be invoked by the nori contract
     require(
       msg.sender == address(_bridgedPolygonNori),
       "FIFOMarket: This contract can only receive BridgedPolygonNORI"
     );
 
-    uint256[] memory ids = new uint256[](_queueLength());
-    uint256[] memory amounts = new uint256[](_queueLength());
-    address[] memory suppliers = new address[](_queueLength());
+    uint256 numberOfRemovals = 0;
```

Comment:
Loop through the array of removals just to find the right size for the arrays above.